orca-core/src/main/groovy/com/netflix/spinnaker/orca/pipeline/persistence/jedis/JedisExecutionRepository.groovy

```diff
@@ -49,12 +48,12 @@ import static java.lang.System.currentTimeMillis
 
 @Component
 @Slf4j
-@CompileStatic
```

Comment:
That's fair ... no need to throw the baby out with the bath water. I'll add it back.